ractive-koa/app.js

```diff
@@ -0,0 +1,136 @@
+'use strict';
+
+// - - - - - - - - - - - - - - - - - - - - - - - - >>>>> vars
+
+var ss = require('socketstream');
+var ssJade = require('ss-jade');
+var ssStylus = require('ss-stylus');
+var koa = require('koa');
+var connect = require('koa-connect');
```

Comment:
`koa-connect` helps mounting express-like middleware to Koa generators. There may be a better tool, but this seems to actually work.